extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -13,18 +13,22 @@ class ArticleComment {
 	const CACHE_VERSION = 2;
```

Comment:
There are a lot of spacing fixes introduced in this class via the stylize.php script.  I also changed array() => [].  Sorry for the length ...